CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Icon: Replace existing filter icon (#565)
 - Contents/Controller: Replace componentWillReceiveProps with UNSAFE_componentWillReceiveProps (#566)
 - Icon: Add new replace/scale icons (#567)
+- defaultProps: Remove the last React defaultProps and transfer it to the already existing but empty ES6 static defaultProps on the same file (#568)
```

Comment:
Even though the motivation here was to modify `defaultProps`, you specifically edited the `ScrollFetch` component. If I were writing the comment, I think I would've called that out as the header of this item.